Fix crash

kyle4269 opened the pull request:
My mistake sorry!

ljmerza commented:
what error are you seeing without these? i took them out because i didnt see why they were needed. you can make a local score var in the functions and pass them back to the global one

kyle4269 commented:
> what error are you seeing without these? i took them out because i didnt see why they were needed. you can make a local score var in the functions and pass them back to the global one

I'll have to replicate it again to get the error. 

But the reason I added the global score, was under set_sublabel I added percentscore = "{:.1%}".format(score).

LOGGER.info(f"Sublabel set successfully to: {sublabel} with {percentscore} confidence")

ljmerza commented:
fixed in v1.7.1

kyle4269 commented:
Awesome! Thank you for fixing that. Makes much more sense to do it your way!
